docs/get-started/deployment-and-usage.md

```diff
@@ -99,6 +99,65 @@ this approach does not guarantee running once on every node. For example,
 tainted, non-ready nodes or some other reasons in Job scheduling may cause some
 node(s) will run extra job instance(s) to satisfy the request.
 
+### Deployment with Helm
+
+Node Feature Discovery Helm chart allow to easily deploy and manage NFD.
+
+#### Prerequisites
+
+[Helm package manager](https://helm.sh/) should be installed.
+
+#### Deployment with Helm
+
+To install the chart with the release name node-feature-discovery:
+
+```bash
+git clone https://github.com/kubernetes-sigs/node-feature-discovery/
+cd node-feature-discovery/deployment
+helm install node-feature-discovery ./node-feature-discovery/
+```
+
+The command deploys Node Feature Discovery on the Kubernetes cluster in the default configuration.
+The Configuration section describes how it can be configured during installation.
+
+#### Configuration
+
+You can override values from `values.yaml` and provide a file with custom values:
+
+```bash
+helm install node-feature-discovery ./node-feature-discovery/ -f <path/to/custom/values.yaml>
+```
+
+To specify each parameter separately you can provide them to helm install command:
+
+```bash
+helm install node-feature-discovery ./node-feature-discovery/ --set nameOverride=NFDinstance --set master.replicaCount=2
+```
+
+#### Uninstalling the Chart
+
+To uninstall the `node-feature-discovery` deployment:
+
+```bash
+helm uninstall node-feature-discovery
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+#### Chart Parameters
+
+In order to tailor the deployment of the Node Feature Discovery to your cluster needs
+We have introduced the following Chart parameters.
+
+| Name | Type | Default | description |
+| ---- | ---- | ------- | ----------- |
+| `image.repository` | string | `gcr.io/k8s-staging-nfd/node-feature-discovery` | NFD image repository |
+| `image.pullPolicy` | string | `Always` | Image pull policy |
+| `master.*` | dict |  | NFD master deployment configuration |
+| `worker.*` | dict |  | NFD master daemonset configuration |
+| `worker.config` | string | `` | NFD worker service configuration |
```

Comment:
This is really good 👍  I think we should add `version` here as well.

In addition, could we open up the `master.*` and `worker.*` parameters, too(?)

Reply:
@marquiz do we need to add all master.* and worker.* params?

Reply:
master.* and worker.*  contain general kubernetes  settings and they are not NFD-specific. That's why I didn't add details to them

Reply:
I think it would be very useful for the reader (like me) to read here what aspects of the deployment are configurable via Helm parameters. Or is it common Helm practice/knowledge that these (and only these) general deployment settings are configurable via parameters of the same name? If so, a link to this information would be nice.